packages/core/test/acceptance/profiler_spec.ts

```diff
@@ -0,0 +1,241 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ProfilerEvent, setProfiler} from '@angular/core/src/render3/profiler';
+import {TestBed} from '@angular/core/testing';
+import {expect} from '@angular/core/testing/src/testing_internal';
+import {onlyInIvy} from '@angular/private/testing';
+
+import {Component, ErrorHandler, EventEmitter, OnInit, Output, ViewChild} from '../../src/core';
+
+
+onlyInIvy('Ivy-specific functionality').describe('profiler', () => {
+  class Profiler {
+    profile() {}
+  }
+
+  let profilerSpy: jasmine.Spy;
+
+  beforeEach(() => {
+    const profiler = new Profiler();
+    profilerSpy = spyOn(profiler, 'profile').and.callThrough();
+    setProfiler(profiler.profile);
+  });
+
+  describe('change detection hooks', () => {
+    it('should call the profiler for creation and change detection', () => {
+      @Component({selector: 'my-comp', template: '<button (click)="onClick()"></button>'})
+      class MyComponent {
+        onClick() {}
+      }
+
+      TestBed.configureTestingModule({declarations: [MyComponent]});
+      const myComp = TestBed.createComponent(MyComponent);
```

Comment:
I think it's a bit more conventional in acceptance tests to have a `fixture` const:
```suggestion
      const fixture = TestBed.createComponent(MyComponent);
      const myComp = fixture. componentInstance;
```